beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -5146,49 +5205,39 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
 
         let blobs_verification_timer =
             metrics::start_timer(&metrics::BLOCK_PRODUCTION_BLOBS_VERIFICATION_TIMES);
-        let maybe_sidecar_list = match (blobs_opt, proofs_opt) {
-            (Some(blobs_or_blobs_roots), Some(proofs)) => {
+        let blob_item = match (blobs_opt, proofs_opt) {
```

Comment:
```suggestion
        let blob_items = match (blobs_opt, proofs_opt) {
```

Another occurrence at L5260 below